### DISCUSSION TITLE
How to use deeplink app to pull up apps

### DISCUSSION THREAD

**Discussion by wds1181977** [WalletConnect/walletconnect-monorepo#1317] [Developer Support]
I see metamask wallet deeplink like this
https://metamask.app.link/wc?uri=wc%3A913aa934-d6a1-473c-ac27-d49a8e9559bd%401%3Fbridge%3Dhttps%253A%252F%252Fi.bridge.walletconnect.org%26key%3D501feb5b2bff1265f46c9c38f60929070c593c15b319b4e8b7189b62cd2907ac

Do we need to make our own set of deeplink generators?   like https://metamask.app.link/

**Comment by chadyj** [comment]
Use the Explorer API to fetch deep links for wallets
https://docs.walletconnect.com/2.0/introduction/cloud-explorer#cloud-explorer-api

For example: ` curl https://explorer-api.walletconnect.com/v1/wallets` will show all wallets.

The part you want looks like this: 
```json
"mobile":{"native":"metamask:","universal":"https://metamask.app.link"},
```
